@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -834,6 +834,21 @@ export class TileGeometryCreator {
                                 unitFactor *
                                 0.5;
 
+                            if (
+                                "outlineWidth" in technique &&
+                                typeof technique.outlineWidth === "number" &&
```

Comment:
If this property is static, you should move this `lineMaterial.outlineWidth` assignment out from update function.
Otherwise it's a wrong way to update interpolated property, because it's not a number, and to check that you should use `isInterpolatedProperty`.

Reply:
@okanishchev Please explain how do you see the usage of `isInterpolatedProperty` here

Reply:
First of all, the technique passed here can't be changed, and all the properties are final. So you can check `isInterpolatedProperty(outlineWidth)` outside of this update function, and if it's false - you only need to set it once, and avoid setting it each frame here. And vice versa - set the value every frame if the condition was true.

Reply:
You still miss the interpolated outlineWidth case

Reply:
@okanishchev fixed.